packages/components/typography/src/typography.css

```diff
@@ -0,0 +1,45 @@
+/* General........................................................ */
+
+:host [data-variant='default'] {
+	font-size: var(--ts-fontsize);
+}
+
+:host [data-variant='title'] {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this usage of the experimental "host" pseudo. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=css%3Aexperimental-pseudo-usage)